.github/actions/send-slack-notification/action.yml

```diff
@@ -43,13 +46,19 @@ runs:
       shell: bash
       env:
         EVENT_NAME: ${{ github.event_name }}
+        BLOCK_MESSAGE: ${{ inputs.message }}
       run: |
-        if [ $EVENT_NAME == 'pull_request' ]
-        then
-          echo "::set-output name=result::${{ github.event.pull_request.title }}"
-        else
-          echo "::set-output name=result::${{ github.event.head_commit.message }}"
-        fi
+        case $EVENT_NAME in
+          pull_request)
+            echo "::set-output name=result::${BLOCK_MESSAGE:-${{ github.event.pull_request.title }}}"
+            ;;
+          issues)
+            echo "::set-output name=result::${BLOCK_MESSAGE:-${{ github.event.issue.body }}}"
+            ;;
+          *)
+            echo "::set-output name=result::${BLOCK_MESSAGE:-${{ github.event.head_commit.message }}}"
+            ;;
+        esac
```

Comment:
would work as intended anyway, but I think it's more clear if before the switch we just check if BLOCK_MESSAGE is not empty and just use it